cmd/docker/daemon_unix.go

```diff
@@ -24,6 +24,7 @@ func newDaemonCommand() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runDaemon()
 		},
+		Deprecated: "and will be removed in docker 1.16. Please run `dockerd` directly.",
```

Comment:
It's really pedantic, but shouldn't this be `removed in Docker 1.16.` (capital D)? :innocent: